api_app/api/routes/workspaces.py

```diff
@@ -234,3 +234,10 @@ async def delete_user_resource(workspace_id: str, service_id: str, resource_id:
     await send_uninstall_message(user_resource, user_resource_repo, previous_deletion_status, ResourceType.UserResource)
 
     return UserResourceIdInResponse(resourceId=user_resource.id)
+
+
+@user_resources_router.patch("/workspaces/{workspace_id}/workspace-services/{service_id}/user-resources/{resource_id}", response_model=UserResourceInResponse, name=strings.API_UPDATE_USER_RESOURCE)
```

Comment:
workspace_id: str, service_id: str, resource_id: str, won't be needed once we have the patch in PR merged so might as well remove it from here so we don't leave it in by mistake when we merge